pyproject.toml

```diff
@@ -74,7 +74,7 @@ before-test = [
   "PIP_CONSTRAINT= pip install PyYAML",
 ]
 test-requires = "-r requirements/test.txt"
-test-command = "pytest -v --no-cov {project}/tests"
+test-command = 'pytest -v -m "not hypothesis" --no-cov {project}/tests'
```

Comment:
@bdraco I thought we'd do this granularly, using an override that selects musl..

Reply:
Anything that is using qemu nearly an order of magnitude slower, and since the hypothesis are not deterministic their performance profile changes between executions and sometimes its even worse so I figure since we already run them on all the test platforms, running them during the wheel builds as well wasn't strictly needed.

Reply:
I think we could still run them on non-emulated  platforms since it extends our environment coverage. But that's something I can look into later. No rush.